### ISSUE TITLE
ATM-6 (1.6.13) **ON SERVER**: Plant Sower from Industrial Foregoing not working at all

### ISSUE THREAD

**Issue by BroodjeKroepoek** [AllTheMods/ATM-6#1931]
ATM-6, Version: 1.6.13

**Describe the bug**
When I put seeds in the Plant Sower it just voids the seeds. I know for sure that I set the machine up correctly. I've even read some bug reports about chunk loading being a potential cause, but that's not the case here.

At first I thought the issue was that I had to hoe the ground en put water down, even though I know the machine should do that on its own. But after trying still nothing happened. I've lost multiple stacks of wheat seeds now by trying to fix this. I even tried breaking the Plant Sower and then replacing after unclaiming the chunks.

The world im playing in is a server. Maybe that has something to do with it?

**To Reproduce**
1. Place Plant Sower down
2. Put range +12 upgrade in the Plant Sower
3. Setup the colored grid squares in the Plant Sower with all wheat seeds
4. Supply power to the Plant Sower
5. Input wheat seeds and watch

Then nothing will happen, but all the seeds are voided.

**Screenshots**
![2021-06-30_04 49 49](https://user-images.githubusercontent.com/85361152/123894591-b7477980-d95e-11eb-98ed-d7ffbf7e4754.png)
Here you can see the Plant Sower with the working area highlighted.

![2021-06-30_04 51 41](https://user-images.githubusercontent.com/85361152/123894701-e78f1800-d95e-11eb-941d-f443c259cc55.png)
Here you can see the setup of the colored squares of the Plant Sower, the range upgrade and that it has power.

**Logs**
No logs. I could provide them I just don't know how.

I hope this can be fixed.

**Comment by BroodjeKroepoek** [comment]
This is still true for version ATM 6 - 1.6.14

**Comment by lowrider001** [comment]
has this been fixed? im running atm6 1.8.3... or is there a biome restriction maybe? all my sower will do is plant tree saplings.. it is not in a claimed chunk so i know its not that directly messing with it at least.

**Comment by BroodjeKroepoek** [comment]
At some point it started working again. Claimed chunks we're not the issue I believe. I also don't exactly know in which version of ATM6 it started working again, because at some point I didn't play much of minecraft and the suddenly there were loads of updates to ATM6.

Tl;dr: The planter works again.
